dashed/assets/javascripts/dashboard.js

```diff
@@ -89,8 +89,8 @@ var Dashboard = function (dashboardData) {
       dashboard = this;
       var gridster = $(".gridster ul").gridster({
         autogrow_cols: true,
-        widget_margins: [10, 10],
-        widget_base_dimensions: [100, 100],
+        widget_margins: [5, 5],
```

Comment:
I think more separation between charts looks significantly better / set it to 10 intentionally. see what @michellethomas thinks